Dockerfile

```diff
@@ -36,13 +36,16 @@ RUN cd /app/tools/license; mkdir jars; cd jars; \
 # stage
 FROM eclipse-temurin:11-jre-jammy
 ENV FLINK_HOME=/opt/flink
-ENV FLINK_PLUGINS_DIR=/opt/flink/plugins
+ENV FLINK_PLUGINS_DIR=$FLINK_HOME/plugins
 ENV OPERATOR_VERSION=1.5-SNAPSHOT
 ENV OPERATOR_JAR=flink-kubernetes-operator-$OPERATOR_VERSION-shaded.jar
 ENV AUTOSCALER_JAR=flink-kubernetes-operator-autoscaler-$OPERATOR_VERSION.jar
 ENV WEBHOOK_JAR=flink-kubernetes-webhook-$OPERATOR_VERSION-shaded.jar
 ENV FLINK_KUBERNETES_SHADED_JAR=flink-kubernetes-standalone-$OPERATOR_VERSION-shaded.jar
 
+ENV USER_DEPENDENCIES_DIR=$FLINK_HOME/dependencies
+RUN mkdir -p $USER_DEPENDENCIES_DIR
```

Comment:
Why not use the same convention as Flink's application mode and name the directory `usrlib`?

Reply:
I did not want to mix up user jar located under 'userlib' in Flink application and user dependencies on Flink operator.

Reply:
Mm I don't have a strong opinion either way, if you think `usrlib` should have different semantics, I think that's fine.

Reply:
@gyfora  what do you think about this?

Reply:
I think this is not usrlib, the operator is controlled by the platform not the user. We could even call it `$FLINK_HOME/operator-lib`

Reply:
agree. changed it.